app/src/main/java/Fragments/HomeFragment.java

```diff
@@ -139,19 +136,21 @@ public boolean onQueryTextSubmit(String query) {
 				}
 
 			    @Override
+				// Responsible for displaying all possible string from the list based on each additionnal character input made by user
 			    public boolean onQueryTextChange(String newText) {
```

Comment:
Are you aware that you changed the functionality of the search we had from only applying the search filter when the user is done to actively scanning through every item EVERY time a change is made? 

The changes you made are inherently voiding the "onQueryTextSubmit" functionality above.

Reply:
Taking note of that 👍 . Will make necessary changes next sprint